go/vt/vttablet/tabletconntest/fakequeryservice.go

```diff
@@ -34,6 +34,37 @@ import (
 	vtrpcpb "vitess.io/vitess/go/vt/proto/vtrpc"
 )
 
+const (
```

Comment:
All these constants are used in the function immediately following their definition, which makes it more readable. So, I believe these should be moved back to where they were before.

Exception: if you see a constant used in more than one place, those can stay here.